tests/browser/environment.js

```diff
@@ -174,7 +185,7 @@ class PuppeteerEnvironment extends JestNodeEnvironment {
                 ws.on('close', () => sockets.delete(ws));
             });
 
-            function sendToPopup(message) {
```

Comment:
`sendToPopup()` currently broadcasts every message to every socket for simplicity. I don't think it's worth making it smarter yet.

Reply:
If it works, don't touch it ;)